spec/factories/profiles.rb

```diff
@@ -1,6 +0,0 @@
-FactoryBot.define do
```

Comment:
We don't really need this anymore. User will automatically have attached profiles and profiles without users don't make sense. If you need a profile in a spec, `create(:user).profile` is the way to go.